ci/Jenkinsfile.android

```diff
@@ -102,14 +106,12 @@ pipeline {
       }
     }
     stage('Cleanup') {
-      steps {
-        sh 'make watchman-clean'
-        sh 'make clean'
-      }
+      steps { sh 'make clean' }
     }
   }
   post {
     success { script { load('ci/github.groovy').notifyPR(true) } }
     failure { script { load('ci/github.groovy').notifyPR(false) } }
+    always { sh 'make _fix-perms' }
```

Comment:
I did, but I still need to copy the read-only version from Nix store into `node_modules.tmp`, which then gets moved into `node_modules`. If the build is stopped while that is happening, we'll end up with read-only `node_modules.tmp`, so better to have this to avoid having to fix things manually every now and then.